build/check-style.sh

```diff
@@ -133,33 +134,39 @@ TestVet() {
     printFuncsStr=$(echo $printFuncsStr$i)
   done
   local vet=$(go tool vet -all -shadow -printfuncs $printFuncsStr . 2>&1)
-  ! (echo "$vet" | grep -vE 'declaration of "?(pE|e)rr"? shadows' | grep -vE '\.pb\.gw\.go:[0-9]+: declaration of "?ctx"? shadows' | grep -vE '^vet: cannot process directory \.git')
+  ! (echo "$vet" | grep -vE 'declaration of "?(pE|e)rr"? shadows' | grep -vE '\.pb\.gw\.go:[0-9]+: declaration of "?ctx"? shadows'
 }
 
 TestGolint() {
-  ! ((golint "$PKG" || true) | grep -vE '((\.pb|\.pb\.gw|embedded|_string)\.go|sql/parser/(yaccpar|sql\.y):)')
+  local golint=$(golint "$PKG")
+  ! echo $golint | grep -vE '((\.pb|\.pb\.gw|embedded|_string)\.go|sql/parser/(yaccpar|sql\.y):)'
```

Comment:
you need quotes (`"$golint"`) or multiple lines become a single line. Same everywhere else below